webapp/store/views.py

```diff
@@ -73,17 +73,24 @@ def store_view():
 
         try:
             featured_snaps_results = api.get_searched_snaps(
-                snap_searched="", category="featured", size=24, page=1
+                snap_searched="", category="featured", size=10, page=1
             )
         except ApiError:
             featured_snaps_results = []
 
         featured_snaps = logic.get_searched_snaps(featured_snaps_results)
 
+        if len(featured_snaps) > 0 and featured_snaps[0]["icon_url"] == "":
+            featured_snaps = featured_snaps[:-1]
```

Comment:
Not sure exactly what happens here... If first snap doesn't have an icon we remove it?
What if second one doesn't have icon as well?

Reply:
The first snap will have the "featured" banner, unless it doesn't have an icon. By default we get 10 snaps (1 main, then 3 rows of 3). 
If we don't show the first snap as a banner - we don't want to have a hanging 10th snap in the grid, so this removes it.